sdk/identity/identity/review/identity.api.md

```diff
@@ -145,15 +146,28 @@ export function getDefaultAzureCredential(): TokenCredential;
 
 export { GetTokenOptions }
 
+// @public
+export interface InteractiveBrowserAuthenticateOptions {
+    abortSignal?: AbortSignalLike;
+    scopes?: string | string[];
+}
+
+// @public
+export type InteractiveBrowserAuthenticationFlow = "implicit-grant" | "auth-code";
+
 // @public
 export class InteractiveBrowserCredential implements TokenCredential {
     constructor(options?: InteractiveBrowserCredentialOptions);
+    authenticate(_options: InteractiveBrowserAuthenticateOptions): Promise<AuthenticationRecord | undefined>;
```

Comment:
Did you mean to include the changes to add the user auth api in this PR as well? If so, would it be cleaner to separate them out into a separate PR or do you think it's too much overhead?

Reply:
I'm adding this as a bridge, since I figured I could re-use this authenticate method during `acquireToken`. Here's an example: https://github.com/Azure/azure-sdk-for-js/pull/13263/files#diff-77d390696993ea049efa74fcaefca2a144ce2671190580d45cf19a84540a292cR203

It's not intended to be related to user-auth, but seemed reasonable as a bridge, if that makes sense.

Reply:
It's not intended to be for user-auth, but I'm trying to build a bridge to get there. I don't know if that makes sense. For this PR, it "exposes" this authenticate method so it can be called independently of the client pipeline. Here's the implementation: https://github.com/Azure/azure-sdk-for-js/pull/13263/files#diff-0ca957af3a0c81d04ae50e6d65cae7fbbd1ee3ed82bf95f3318039a0644f5dd6R106

Reply:
After offline discussion it seems the semantics of this method might slightly different from what we intend to add with our extended user authentication apis. For now let's hold off on adding `authenticate` and `InteractiveBrowserAuthenticateOptions` to the public API.

Reply:
For sure! Thank you 🌞